Backend/app/routes/ai_query.py

```
@@ -0,0 +1,244 @@
+                    from ..routes.brand_dashboard import get_creator_profile
+                    api_result = await get_creator_profile(**api_args)
+                elif intent == "analytics_performance":
+                    from ..routes.brand_dashboard import get_campaign_performance
+                    api_result = await get_campaign_performance(**api_args)
+                elif intent == "analytics_revenue":
+                    from ..routes.brand_dashboard import get_revenue_analytics
+                    api_result = await get_revenue_analytics(**api_args)
+                elif intent == "contracts":
+                    from ..routes.brand_dashboard import get_brand_contracts as get_contracts
+                    api_result = await get_contracts(**api_args)
+            except Exception as api_exc:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)